lib/Game.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "Player"
+require_relative "Board"
+
+class Game
+    def initialize
+        @board =  Board.new
+
+        @player_1 = Player.new("Player 1", "X", @board)
```

Comment:
Same thing here, can't we just pass instantiated player1 and player2 objects as arguments in creating a Game object?